### ISSUE TITLE
Plots are not accessible to screen readers

### ISSUE THREAD

**Issue by hturner** [r-wasm/webr#272]
Comment from @ajrgodfrey

> Plots are not accessible to the screen reader - should be able to select and hear that a plot has been created in the plot pane 

Maybe a plot number could be included in the alt text.

**Comment by georgestagg** [comment]
As of 7df7964 plot images contain a label with the plot number included, e.g. `aria-label="R Plot 2"`.

I found that although plots can't be selected with Tab navigation, I was able to navigate to and hear the image label using macOS VoiceOver, so hopefully screen readers should now be able to read the label too.

(It will take a few hours for the webR website to update, but the changes should be available by tomorrow.)
